analyzer/uncategorized/evm_raw_tx_test.go

```diff
@@ -130,29 +137,37 @@ func TestDecodeBasic(t *testing.T) {
 	decodeExpectCall(
 		t,
 		"f86b8085e8d4a510008227109413978aee95f38490e9769c39b2773ed763d9cd5f872386f26fc10000801ba0eab47c1a49bf2fe5d40e01d313900e19ca485867d462fe06e139e3a536c6d4f4a014a569d327dcda4b29f74f93c0e9729d2f49ad726e703f9cd90dbb0fbf6649f1",
-		0,
-		"13978aee95f38490e9769c39b2773ed763d9cd5f",
-		10_000_000_000_000_000,
-		"",
-		10_000,
-		1_000_000_000_000,
-		// "cow" test account
-		"cd2a3d9f938e13cd947ec05abc7fe734df8dd826",
-		0,
+		&callDetails{
+			commonDetails: commonDetails{
+				chainID:  0,
+				value:    10_000_000_000_000_000,
+				gasLimit: 10_000,
+				gasPrice: 1_000_000_000_000,
+				// "cow" test account
```

Comment:
lol what's cow?

Reply:
Yeah, have I been doing it wrong all this time with Alice/Bob? 🤔

Reply:
https://github.com/oasisprotocol/oasis-sdk/blob/5aa39a2726f196a7b7c08e322ab27bf91f2670d8/runtime-sdk/modules/evm/src/raw_tx.rs#L304 ->
https://github.com/ethereum/tests/blob/v10.0/BasicTests/txtest.json ->
https://github.com/ethereum/tests/blob/v10.0/BasicTests/keyaddrtest.json#L5

saw it in a test from ethereum

interestingly, it has had quite some activity on Ethereum https://etherscan.io/address/0xcd2a3d9f938e13cd947ec05abc7fe734df8dd826